tests/examples/hello/src/iota.cc

```diff
@@ -0,0 +1,28 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class IotaTask : public Task<IotaTask, IOTA> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &output = context.outputs()[0];
```

Comment:
IMHO it would be better for didactic purposes if we spelled out the types here.

Reply:
Done - except for the accessor. That's messy enough that I kept it `auto`